setup.cfg

```diff
@@ -1,2 +1,43 @@
+[metadata]
+name = fairlearn
+version = attr: fairlearn.__version__
+author = Miroslav Dudik
```

Comment:
Time to adjust that? I think this can also have authors instead of singular author?